packages/backend/package.json

```diff
@@ -25,6 +25,7 @@
     "cors": "^2.8.5",
     "express": "^4.21.1",
     "node-cron": "^3.0.3",
-    "seedrandom": "^3.0.5"
+    "seedrandom": "^3.0.5",
+    "sqlite3": "^5.1.7"
```

Comment:
I may run into some issues around deploying this in a container - especially on the pi, because I think it uses some udnerlying c++ precompiled libs.  So that'd be specific to the arch, and may need specific packaging.